src/main/java/com/orbitz/consul/cache/CacheDescriptor.java

```diff
@@ -0,0 +1,32 @@
+package com.orbitz.consul.cache;
+
+public class CacheDescriptor {
```

Comment:
I don't get what mean the fields of this class. Would you add comments for explain its use?

Reply:
Done.